packages/terra-consumer-nav/package.json

```diff
@@ -36,8 +36,14 @@
     "terra-button": "^1.1.0",
     "terra-icon": "^1.0.0",
     "terra-mixins": "^1.0.0",
+    "terra-modal": "^1.2.0",
```

Comment:
Did not use terra-modal, will remove it. UX callout requires the modal title to be on the overlay, so create this modal component using terra-overlay. It will be replaced when Terra-consumer has its own Modal.